sdks/python/apache_beam/testing/load_tests/pardo_test.py

```diff
@@ -125,6 +155,70 @@ def process(self, element):
         'Measure time: End' >> beam.ParDo(MeasureTime(self.metrics_namespace)))
 
 
+class StatefulLoadGenerator(beam.PTransform):
```

Comment:
Nice to see that it is finally possible to achieve this with timers! Should `GenerateLoad` be reusable?

Reply:
Potentially, yes. The implementation should be changed to use SDF eventually but that is not properly supported yet for Python.